app/components/collections/edit/license_component.html.erb

```diff
@@ -0,0 +1,23 @@
+<fieldset>
+  <div class="form-check d-flex pt-2">
+    <div class="ps-2">
+      <%= form.radio_button :license_option, 'required', class: 'form-check-input' %>
+      <%= form.label :license_option_required, 'Require license for all deposits', class: 'form-check-label' %>
+    </div>
+    <div class="ps-2">
+      <%= render Elements::Forms::SelectFieldComponent.new(form:, field_name: :license, options: license_options, required: true, hidden_label: true, label:, width: 250, container_classes: 'mb-4', label_classes: 'h5') %>
```

Comment:
Can you not set required, as that will result in client side validation?

Why is the label_classes set to `h5`?

Reply:
re: 'h5' - copy pasta, I missed that was getting set.